h2o-py/tests/testdir_algos/automl/pyunit_PUBDEV_8077_automl_args_tomaf.py

```diff
@@ -0,0 +1,33 @@
+from __future__ import print_function
+import sys, os
+sys.path.insert(1, os.path.join("..","..",".."))
+import h2o
+from tests import pyunit_utils
+from h2o.automl import H2OAutoML
+
+"""
+This test is from Tomas Fryda.  He discovered that if GLM alpha array is specified to be an array, the following test
+will crash.  I fixed the bug and this test should run to completion.  No assert statement is needed for this test.
+"""
+max_models = 2
+
+def automl_tomf():
```

Comment:
This test is dependent on a setting in stacked ensemble that isn't the default yet and so this test would be useless if the `_alpha` is not set in `SE`.

Normally, I would ask you if you could make a test that doesn't depend on AutoML/SE but since the fix is so simple, I think we don't need to add a test for this (although it would be nice to have bigger test coverage for all the algos).

Reply:
@tomasfryda I just added 3 tests to test that alpha array works when generate_scoring_history is on or off.